Extension/package.json

```diff
@@ -1353,6 +1353,16 @@
                     "type": "boolean",
                     "description": "%c_cpp.debuggers.logging.engineLogging.description%",
                     "default": false
+                  },
+                  "threadExit": {
+                    "type": "boolean",
+                    "description": "%c_cpp.debuggers.logging.threadExit.description%",
+                    "default": false
+                  },
+                  "processExit": {
```

Comment:
Can the name be more descriptive like threadExitLogging, processExitLogging?

Reply:
I don't think that would be good since it would be redundant, because it would look like:
```
"logging": {
   "threadExitLogging": true
   "processExitLogging": false
}
```

Reply:
Ah, okay. I was looking at engineLogging and forgot about the surrounding "logging".